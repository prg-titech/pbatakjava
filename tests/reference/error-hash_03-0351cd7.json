{
    "basename": "error-hash_03-0351cd7",
    "cmd": "{infile}",
    "infile": "tests/hash_03/build.sh",
    "infile_hash": "07d77b2fc7fb8d2b1021cb92768af671c7f4ead885a0751bb8be44a5",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-hash_03-0351cd7.stderr",
    "stderr_hash": "01c6e741f417b6a1d96127d4297c20a7a01e0c5c46496ce26ad68389",
    "returncode": 4
}