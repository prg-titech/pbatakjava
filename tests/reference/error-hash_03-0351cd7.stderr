tests/hash_03/Test.java:13: error: no applicable method for exists(digest2)
tests/hash_03/Test.java:15: error: no applicable solution for Dir«1,2»
tests/hash_03/Test.java:15: error: no applicable solution for Dir«1,2»
